src/info/author.rs

```diff
@@ -77,12 +77,25 @@ impl std::fmt::Display for AuthorsInfo {
         let mut authors_info = String::new();
 
         let pad = self.title().len() + 2;
+        let mut alignment_pad: usize = 0;
 
         for (i, author) in self.authors.iter().enumerate() {
             if i == 0 {
                 write!(authors_info, "{author}")?;
             } else {
-                write!(authors_info, "\n{:<width$}{}", "", author, width = pad)?;
+                // Set padding based on top author
+                if alignment_pad == 0 {
+                    alignment_pad = author.to_string().split_once('%').unwrap().0.len();
+                }
+                let author_pad =
+                    alignment_pad - author.to_string().split_once('%').unwrap().0.len();
```

Comment:
This could probably be simplified by using `str::find`.

Reply:
Ah, great idea. Made the change.